script/gen_requirements_all.py

```diff
@@ -17,7 +17,16 @@
 from homeassistant.util.yaml.loader import load_yaml
 from script.hassfest.model import Integration
 
-COMMENT_REQUIREMENTS = (
+
+def normalize(req: str) -> str:
+    """Normalize a requirement."""
+    return req.lower().replace("_", "-")
+
+
+# Requirements which can't be installed on all systems because they rely on additional
+# system packages. Requirements listed in EXCLUDED_REQUIREMENTS_ALL will be commented-out
+# in requirements_all.txt and requirements_test_all.txt.
+EXCLUDED_REQUIREMENTS_ALL = (
     "atenpdu",  # depends on pysnmp which is not maintained at this time
```

Comment:
Move `INCLUDED_REQUIREMENTS_WHEELS` before this variable, so we can use it and reduce duplicated code

```suggestion
EXCLUDED_REQUIREMENTS_ALL = (
    *INCLUDED_REQUIREMENTS_WHEELS,
    "atenpdu",  # depends on pysnmp which is not maintained at this time
```